vmm/src/api/http/http_endpoint.rs

```diff
@@ -442,7 +238,13 @@ impl PutHandler for VmAddNet {
     ) -> std::result::Result<Option<Body>, HttpError> {
         if let Some(body) = body {
             let mut net_cfg: NetConfig = serde_json::from_slice(body.raw())?;
-            attach_fds_to_cfg(files, &mut net_cfg)?;
+            // Need to clone, as files in req.files are closed on drop.
+            let fds: Vec<i32> = files
+                .iter()
+                .map(|f| f.try_clone().unwrap().into_raw_fd())
+                .collect();
+
+            net_cfg.consume_fds(fds);
```

Comment:
`attach_fds_to_cfg()` did some sanity checks and helpful logging, which you removed now. This is a little unfortunate and I think we can do better here.

Reply:
We need to handle error from `consume_fds()`, the same as what's being done for `VmCreate`.

Also, I agree the (rest of) sanity checks and helpful logging should be added to `consume_fds()` if applicable.